### PR TITLE
bidi plugin

### PR DESCRIPTION
**Pull request by TxHawks**
This PR add a plugin to enable flow-direction independent style authoring, as discussed in #379

### CODE REVIEW

File: packages/fela-plugin-bidi/src/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* @flow */
+import transformStyle from 'bidi-css-js'
+
+import type DOMRenderer from '../../../flowtypes/DOMRenderer'
+import type NativeRenderer from '../../../flowtypes/NativeRenderer'
+
+type Type = 1 | 2 | 3 | 4 | 5
```
Comment (TxHawks) [inline packages/fela-plugin-bidi/src/index.js]: I copied this from `fela-plugin-rtl`, but can't say I understand why the type definitions are there

Comment (rofrischmann) [reply packages/fela-plugin-bidi/src/index.js]: Hehe. And I copied that from another plugin that directly uses the renderer. Feels like we can remove those :)

**Comment by TxHawks** [comment]
By the way, there is a bit of a messed up situation here regarding plugin order.
This plugin obviously does not know how to handle arrays of values, such as the ones passed to `fela-plugin-fallback-values`, and therefore has to be passed to the `plugins` array after `fela-plugin-fallback-values`. 
This, on its own, isn't an issue - however, `fela-plugin-prefixer` must be passed to the `plugins` array _before_ `fela-plugin-fallback-values`, meaning rules transformed by `fela-plugin-bidi` will not be prefixed.

I think this probably isn't a problem in the real world, because I don't think any of the values handled by this plugin should ever be prefixed, but there might be something I'm not thinking about.

This is also true for `fela-plugin-rtl`

**Comment by TxHawks** [comment]
@rofrischmann - does this look fine now, it is there anything you'd like me to change?

**Comment by rofrischmann** [comment]
Looks pretty cool, but I am too busy to review that today. Let me check back tomorrow =)

**Comment by rofrischmann** [comment]
Looks great for me! Will merge and release right away :)

**Comment by TxHawks** [comment]
Thanks!

**Pull request merged**